libs/car_model.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, name, battery_power, abrp_name=None, reg=None,
     ElecModel("Mokka-e", 46, "opel:corsae:20:50", r"VXKUKZKX.*"),  # VXKUKZKXZM
     ElecModel("Zaphira-e", 68, "peugeot:etraveler:21:75:opel", r"VXEVZZKX.*"),  # VXEVZZKXZMZ
     ElecModel("E-C4", 46, "citroen:ec4:21:50", r"VR7BCZKX.*"),  # VR7BCZKXCM
+    ElecModel("DS3 Crossback e-tense", 46, "ds:3crossback:20:48", r"VR1UJZKXZL*"),  # VR1UJZKXZL
```

Comment:
the regex should end with `.*`

Reply:
Corrected in another pull request
I failed to add my new commit to this pull request
Sorry... Hope this is OK now